public/content/deprecated-software/index.md

```diff
@@ -21,12 +21,6 @@ This section is for software for the desktop, command line, or server which has
 
 ### OpenEthereum {#open-ethereum}
 
-<p align="center">
```

Comment:
Shouldn't we keep this? not sure if I understand why we are deleting this.

Reply:
The image source isn't supported, so we would either need to configure this or since we don't consistenly use images on this page, standardize around not using images. I chose to just standardize around not using images instead of supporting this image type. I think a broader discussion on the purpose of this page can be had, but scope creep for this.